### ISSUE TITLE
Add "pendingsignature" attribute to InvocationResult

### ISSUE THREAD

**Issue by gsmachado** [neow3j/neow3j#674]
**Summary or problem description**

Add `pendingsignature` attribute to `InvocationResult`.

![telegram-cloud-photo-size-5-6059983376603000322-w](https://user-images.githubusercontent.com/2124276/140328221-83d2224b-3fbc-405b-af27-68591c1686d3.jpg)

![telegram-cloud-photo-size-5-6059983376603000323-x](https://user-images.githubusercontent.com/2124276/140328234-66e2a09f-e46b-4653-ae4b-fbdb679e2a35.jpg)

**What part of neow3j does this apply to?**
- SDK

**Comment by gsmachado** [comment]
@bettybao1209: can you please tell me in which specific condition the `pendingsignature` attribute is set in the returning JSON? I mean, when the `context.Completed` is not set? Can you give us an example?

![image](https://user-images.githubusercontent.com/2124276/140434547-7f1edd58-c109-4de3-bfd7-426e94739a41.png)

**Comment by bettybao1209** [comment]
> @bettybao1209: can you please tell me in which specific condition the `pendingsignature` attribute is set in the returning JSON? I mean, when the `context.Completed` is not set? Can you give us an example?
> 
> Is it when a function is invoked but the wallet is not "opened" to sign the tx?

If we open a wallet with multisig address included in the rpc node, and invokefunction with the signers including the multisig account, then the result should include the pendingsignature.

**Comment by csmuller** [comment]
@bettybao1209 I have trouble receiving the `pendingsignature` attribute from an `invokefunction` call. I have a multi-sig account on the neo-node. The corresponding private keys are not available on the node. The multi-sig account has some NEO. Then i try to do the following transfer away from the multi-sig account. The account is there as a signer.

```json
{
	"jsonrpc": "2.0",
	"method": "invokefunction",
	"params": [
		"0xef4073a0f2b305a38ec4050e4d3d28bc40ea63f5",
		"transfer",
		[
			{
				"type": "Hash160",
				"value": "0x461c7f08218c8bf47349f1576e26cf697a9da4d6"
			},
			{
				"type": "Hash160",
				"value": "0x69ecca587293047be4c59159bf8bc399985c160d"
			},
			{
				"type": "Integer",
				"value": "10"
			},
			{
				"type":"Array",
				"value":[]
			}
		],
		[
			{
				"account": "461c7f08218c8bf47349f1576e26cf697a9da4d6",
				"scopes": "CalledByEntry"
			}
		]
	],
	"id": 3
}
```

From this i expect to get a response containing the `pendingsignature` attribute. But, i don't. What am i doing wrong?

**Comment by bettybao1209** [comment]
@csmuller I think you should sign the tx with one of the private keys for the multi-sig account to generate the pending signature.

**Comment by csmuller** [comment]
Resolved in #677.
